crates/bevy_input/src/keyboard.rs

```diff
@@ -31,206 +46,365 @@ pub fn keyboard_input_system(
+    /// The `Escape` / `ESC` key, next to the `F1` key.
     Escape,
 
+    /// The `F1` key.
     F1,
+    /// The `F2` key.
     F2,
+    /// The `F3` key.
     F3,
+    /// The `F4` key.
     F4,
+    /// The `F5` key.
     F5,
+    /// The `F6` key.
     F6,
+    /// The `F7` key.
     F7,
+    /// The `F8` key.
     F8,
+    /// The `F9` key.
     F9,
+    /// The `F10` key.
     F10,
+    /// The `F11` key.
     F11,
+    /// The `F12` key.
     F12,
+    /// The `F13` key.
     F13,
+    /// The `F14` key.
     F14,
+    /// The `F15` key.
     F15,
+    /// The `F16` key.
     F16,
+    /// The `F17` key.
     F17,
+    /// The `F18` key.
     F18,
+    /// The `F19` key.
     F19,
+    /// The `F20` key.
     F20,
+    /// The `F21` key.
     F21,
+    /// The `F22` key.
     F22,
+    /// The `F23` key.
     F23,
+    /// The `F24` key.
     F24,
 
-    /// Print Screen/SysRq.
+    /// The `Snapshot` / `Print Screen` key.
     Snapshot,
-    /// Scroll Lock.
+    /// The `Scroll` / `Scroll Lock` key.
     Scroll,
-    /// Pause/Break key, next to Scroll lock.
+    /// The `Pause` / `Break` key, next to the `Scroll` key.
     Pause,
 
-    /// `Insert`, next to Backspace.
+    /// The `Insert` key, next to the `Backspace` key.
     Insert,
+    /// The `Home` key.
     Home,
+    /// The `Delete` key.
     Delete,
+    /// The `End` key.
     End,
+    /// The `PageDown` key.
     PageDown,
+    /// The `PageUp` key.
     PageUp,
 
+    /// The `Left` / `Left Arrow` key.
     Left,
+    /// The `Up` / `Up Arrow` key.
     Up,
+    /// The `Right` / `Right Arrow` key.
     Right,
+    /// The `Down` / `Down Arrow` key.
     Down,
 
-    /// The Backspace key, right over Enter.
+    /// The `Back` / `Backspace` key, right over the `Enter` key.
     Back,
-    /// The Enter key.
+    /// The `Return` / `Enter` key.
     Return,
-    /// The space bar.
+    /// The `Space` / `Spacebar` / ` ` key.
     Space,
 
-    /// The "Compose" key on Linux.
+    /// The `Compose` key on Linux.
     Compose,
-
+    /// The `Caret` / `^` key.
     Caret,
 
+    /// The `Numlock` key.
     Numlock,
+    /// The `Numpad0` / `0` key.
     Numpad0,
+    /// The `Numpad1` / `1` key.
     Numpad1,
+    /// The `Numpad2` / `2` key.
     Numpad2,
+    /// The `Numpad3` / `3` key.
     Numpad3,
+    /// The `Numpad4` / `4` key.
     Numpad4,
+    /// The `Numpad5` / `5` key.
     Numpad5,
+    /// The `Numpad6` / `6` key.
     Numpad6,
+    /// The `Numpad7` / `7` key.
     Numpad7,
+    /// The `Numpad8` / `8` key.
     Numpad8,
+    /// The `Numpad9` / `9` key.
     Numpad9,
 
+    /// The `AbntC1` key.
     AbntC1,
+    /// The `AbntC2` key.
     AbntC2,
+
+    /// The `NumpadAdd` / `+` key.
     NumpadAdd,
+    /// The `Apostrophe` / `'` key.
     Apostrophe,
+    /// The `Apps` key.
     Apps,
+    /// The `Asterik` / `*` key.
     Asterisk,
+    /// The `Plus` / `+` key.
     Plus,
+    /// The `At` key.
     At,
+    /// The `Ax` key.
```

Comment:
Same as `At`

Reply:
I also couldn't find anything on the internet. No idea.

Reply:
Yeah, this appears to be `winit's` fault. I'll file an issue.

Reply:
https://github.com/rust-windowing/winit/issues/2265